scripts/generate-constraints.py

```diff
@@ -83,6 +83,9 @@ def newArgparser():
             used = []
             boardPinInfo = {}
             pinName = ''
+            pinFound = len([x for x in projectPinInfo.keys() if x in bindings.keys()])
```

Comment:
This script has gotten a bit unruly, but the idea is that common pins can be specified in the boardinfo file and other ones can be fully specified in the project pinout.json. This change would prevent a pin from being fully specified in the pinout.json file. Please revert this change to generate-constraints.py